kolibri/utils/version.py

```diff
@@ -64,13 +70,16 @@ def derive_version_from_git_tag():
         subprocess.CalledProcessError,  # not a git repo
         OSError  # git executable doesn't exist
     ):
-        return None
+        pass
+    else:
+        version_string = parse_git_tag_version_string(git_describe_string)
 
-    return parse_git_tag_version_string(git_describe_string)
+    os.chdir(current_dir)
```

Comment:
I think it's fair to put it in a context manager -- they ensure that the `__exit__()` function is called despite any exceptions. That should lead to less surprises in case any errors from `check_output` bubbles up and `os.chdir` down below doesn't get called.